### PR TITLE
Doc: add description to README.md

### PR DESCRIPTION
**Pull request by frehberg**

### CODE REVIEW

**Comment by frehberg** [comment]
Issue #9

File: README.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
+# The Real-time Publish-Subscribe Protocol for Rust
+
+This is the implementation of the Real-time Publish-Subscribe Protocol (RTPS) DDS Interoperability Wire Protocol for Rust.
+
+The Data Distribution Service for real-time systems (DDS) is an Object Management Group (OMG) machine-to-machine connectivity framework that aims to enable scalable, real-time, dependable, high-performance and interoperable data exchanges using a publish–subscribe pattern. DDS addresses the needs of applications like air-traffic control, smart grid management, autonomous vehicles, robotics, transportation systems, power generation, medical devices, simulation and testing, aerospace and defense, and other applications that require real-time data exchange [[Wiki]][wiki-dds-url].
+
+### Features
+The objectives of this implementation are:
+* Implementing RTPS according to specification [[RTPS-2.2]][omg-rtps-url]
```
Comment (Klapeyron) [inline README.md]: There is also new beta version (2.3) of RTPS, when it is available, then we will migrate to the newest version

File: README.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+* Integrating into the Rust-Tokio event system for async IO.
+* Feature X
+* Feature Y
+...
+TBD
+
+### Roadmap
+* Version 0.1 Feature X, Feature Y
+* Version 0.2: Feature Z
```
Comment (Klapeyron) [inline README.md]: Could we stay on hold with this section until we migrate to the GitLab IssueBoard?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,23 @@
...
+
+
 [![Crates.io][crates-badge]][crates-url]
```
Comment (Klapeyron) [inline README.md]: Personally I prefer to put badges somewhere at the beginning of the README, could you put them somewhere on the top?

**Comment by frehberg** [comment]
Ok, this might be the initial version of README.md

**Pull request merged**